packages/number-field/src/NumberField.ts

```diff
@@ -340,6 +340,13 @@ export class NumberField extends TextfieldBase {
                     new Event('change', { bubbles: true, composed: true })
                 );
                 break;
+            case 'Enter':
+                event.preventDefault();
+                this.inputElement.value = this.formattedValue;
+                this.dispatchEvent(
+                    new Event('change', { bubbles: true, composed: true })
+                );
```

Comment:
This looks good are there any tests that would be beneficial to add in support of this going forward?